volatility/framework/plugins/windows/poolscanner.py

```diff
@@ -370,6 +380,7 @@ def generate_pool_scan(cls,
 
             mem_object = header.get_object(type_name = constraint.type_name,
                                            use_top_down = is_windows_8_or_later,
+                                           kernel_symbol_table = symbol_table,
```

Comment:
Again, a bit of an API change, I'll let @iMHLv2 comment on the efficacy of this.  If he says it's good, it's good.  5:)

Reply:
See the later comment, but in this plugin it needs the `_required_framework_verison` setting appropriately after that change is made.